### ISSUE TITLE
[Bug] TableView's SwitchCell does not function correctly with screenreaders

### ISSUE THREAD

**Issue by jackama** [xamarin/Xamarin.Forms#10852]
### Description

Using a native screenreader (VoiceOver on iOS; TalkBack on Android), activating a SwitchCell in a settings TableView reads out the current state of the switch (On or Off), but does not toggle it.

On Android, selecting the switch _within_ the cell and activating it does toggle it, but activating the row does not.
On iOS, the SwitchCell cannot be toggled.

### Steps to Reproduce

1. Create a template Xamarin.Forms project 
2. Add a TableView with SwitchCells to a page as per [the official example](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/tableview)
3. Build and deploy to a physical device (iOS or Android)
4. Enable the devices native screenreader
5. Navigate to the page, highlight one of the SwitchCell rows
6. Double Tap to activate

### Expected Behavior

The switch should toggle and the screenreader should announce the new switch state.

### Actual Behavior

The switch **does not** toggle and the screenreader announces the unchanged switch state.
SwitchCell's OnChanged event does not fire.

- Or -

On Android, the switch itself can be highlighted (with difficulty - the interaction area is small) and activating it does toggle the state. **However**, the switch does not utilise any of the SwitchCell's AutomationProperties, so it is unlabelled and unannounced - the screenreader simply annouces "On switch" or "Off switch".

### Basic Information

- Version with issue: Xamarin.Forms 4.5.0.495
- Last known good version: N/A
- IDE: Visual Studio Community 2019
- Platform Target Frameworks: 
  - iOS:  8.0.30703  
  - Android: 9.0 
  - UWP: Not tested
- Nuget Packages:
  - Newtonsoft
  - SkiaSharp
  - Xamarin.Auth
  - Xamarin.Essentials  

### Screenshots

<img src="https://user-images.githubusercontent.com/65961146/82911256-93e41b00-9f63-11ea-8a7e-6deef5eecd8b.png" alt="iOS switch cell" width="200"/>
<img src="https://user-images.githubusercontent.com/65961146/82911268-9777a200-9f63-11ea-9066-1910635a85cf.png" alt="Android switch cell" width="200"/>
<img src="https://user-images.githubusercontent.com/65961146/82911273-99d9fc00-9f63-11ea-9577-46073b3223ce.png" alt="Android switch cell toggle selcted" width="200"/>
